### DISCUSSION TITLE
Algolia - Add a large search box on index page

### DISCUSSION THREAD

**Discussion by ArtFlag** [facebook/docusaurus#6234] [Q&A]
Hi, 

I'd like to add a large search bar on my homepage, essentially the large search bar we get when click in the small search bar in the top-right corner of the default theme.  For example, like what we see on [these docs](https://help.shortcut.com/hc/en-us/categories/201050836-Working-With-Clubhouse).

I haven't managed to import all the correct styles and features of the search bar. Does anybody know how to do this? 

Thanks!

**Comment by slorber** [comment]
It's outside of the scope of Docusaurus

You can implement this yourself by using existing React components that Algolia provides in `src/index.js`

Maybe @shortcuts have any tips
